pkg/liqoctl/status/namespace.go

```diff
@@ -26,55 +25,65 @@ import (
 
 const nsCheckerName = "Namespace existence check"
 
-// namespaceChecker implements the Check interface.
+// NamespaceChecker implements the Check interface.
 // checks if the namespace passed as an argument to liqoctl status command
 // exists. If it does not exist the liqoctl status returns.
-type namespaceChecker struct {
+type NamespaceChecker struct {
 	options       *Options
 	succeeded     bool
 	failureReason error
+	silent        bool
 }
 
-func newNamespaceChecker(options *Options) *namespaceChecker {
-	return &namespaceChecker{
+// NewNamespaceChecker returns a new NamespaceChecker.
+func NewNamespaceChecker(options *Options, silent bool) *NamespaceChecker {
+	return &NamespaceChecker{
 		options: options,
+		silent:  silent,
 	}
 }
 
-func (nc *namespaceChecker) Collect(ctx context.Context) error {
+// Silent implements the Check interface.
+func (nc *NamespaceChecker) Silent() bool {
+	return nc.silent
+}
+
+// Collect implements the Check interface.
+func (nc *NamespaceChecker) Collect(ctx context.Context) {
 	// Check if the namespace exists.
 	if _, err := nc.options.KubeClient.CoreV1().Namespaces().Get(ctx, nc.options.LiqoNamespace, v1.GetOptions{}); err != nil {
 		nc.succeeded = false
 		nc.failureReason = err
-		return nil
+		return
 	}
-
 	nc.succeeded = true
-
-	return nil
 }
 
 // GetTitle returns the title of the checker.
-func (nc *namespaceChecker) GetTitle() string {
+func (nc *NamespaceChecker) GetTitle() string {
 	return nsCheckerName
 }
 
-func (nc *namespaceChecker) Format() (string, error) {
+// Format implements the Check interface.
+func (nc *NamespaceChecker) Format() string {
 	var text string
 	if nc.succeeded {
-		text += nc.options.Printer.Success.Sprint(pterm.Sprintf("%s liqo control plane namespace %s exists",
-			nc.options.Printer.Success.Prefix.Style.Sprint(output.CheckMark),
-			nc.options.Printer.Success.Prefix.Style.Sprint(nc.options.LiqoNamespace)))
-		return text, nil
+		if !nc.silent {
+			text += nc.options.Printer.Success.Sprint(pterm.Sprintf("%s liqo control plane namespace %s exists",
+				nc.options.Printer.Success.Prefix.Style.Sprint(output.CheckMark),
+				nc.options.Printer.Success.Prefix.Style.Sprint(nc.options.LiqoNamespace)))
+		}
```

Comment:
Is it necessary this if? You could always return the text, and discard it if not necessary.